Adding ShipIt XIV blog

amcginn opened the pull request:
![shipit xiv - engineering health](https://user-images.githubusercontent.com/22381097/53988620-8ab22880-40e9-11e9-81c2-76bc1874599c.png)

amcginn commented:
Removing first two images:
![shipit xiv - engineering health](https://user-images.githubusercontent.com/22381097/53990318-9e5f8e00-40ed-11e9-988e-6a986e241989.png)

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+---
+
+Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.
+It seems a bit daring to voluntarily bear the winter weather, stay up for 24-hours, complete a project start to finish, and place all other responsibilities on hold. Nevertheless, ShipIt Day always is bursting with Cerner associates prepared to take on the challenge.  *help this sentence please* Check out the video below to hear from ShipIt Day participants and supporters as to why ShipIt Day has ...
```
> It seems a bit daring to voluntarily bear the winter weather, stay up for 24-hours, complete a project start to finish, and place all other responsibilities on hold.

I think it would be good to exclude this statement. While the time period is over 24 hours, we don't ask people to stay up for 24 hours, nor do we recommend it. It is just that we have the deadline set one day out, use the time you want in that window.

amcginn replied:
Excluded introductory comment. Changed to suggestion below.

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+tags: [culture, engineering, hackathon, shipit]
+---
+
+Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.
```
> Readily awaiting the winter storm racing towards Kansas City, ShipIt Day XIV teams began preparing their ideas and tactics to compete in the 24-hour hackathon competition.

I would avoid this entry statement, as it also seems like something bad is coming, then the statement flexes to our event. Perhaps:

> It is winter time in Kansas City, which also included our fourteenth edition of ShipIt, our routine hackathon competition we host at Cerner Engineering. Although the snowy weather may have kept participants between the warm walls of our Innovations Campus, the winter storm did not disturb the initiative and enthusiasm of the fourteen participating teams.

amcginn replied:
Updated to suggested introduction

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-1.jpg 600px %}
+
+ >“We created "HCCgle" (pronounced "WHO-gull"), a search application for looking up HCC information by ICD-10 diagnosis code (e.g. - "E13.11") or by term (e.g. - "Diabetes mellitus"), including across physician-friendly terminologies like IMO and SNOMED CT. The app presented which HCC categories the requested term belongs to (if any) and shows how its categorization in the HCC specification has c ...
```
Including hyperlinks to terms that would be not obvious to readers:
* IMO -> https://en.wikipedia.org/wiki/Intelligent_Medical_Objects
* SNOMED -> https://en.wikipedia.org/wiki/Systematized_Nomenclature_of_Medicine
* ICD-10 -> https://en.wikipedia.org/wiki/ICD-10

amcginn replied:
Included suggested links

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-2.jpg 600px %}
+
+>“We created an innovation/strategic roadmap prototype for Cerner's clients that shows past, current and future projects, along with crucial data points for each. The data was directly from a Microsoft SQL server that houses all of our PPM project data, and the application was built on Ruby on Rails and React.”– Risky Salt
```
Remove "PPM project data" and use "project management data"

amcginn replied:
Changed wording

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-3.jpg 600px %}
+
+>“Our team worked on Prototype for a Flow Sheet- a real-time, customizable data visualization tool for patient device data. In the real world, this would be used to view trends in a patient’s health to proactively identify risks. The goal of this project was Protoype UI to support a dense flow sheet/graphical view of critical care data that would support iBus and Millennium data sources, as well  ...
```
> Our team worked on Prototype for a Flow Sheet- a real-time, 

Our team worked on a prototype of a real-time flow sheet, that was a customizable data visualization tool for patient device data.

> The goal of this project was Protoype UI to support a dense flow sheet/graphical view of critical care data that would support iBus and Millennium data sources, as well as understand complexity to better gauge bringing function into solution.

The goal of this project was to build a prototype UI support a dense flow sheet and graphical view of critical care data. This would include Cerner [CareAware iBus](https://www.cerner.com/pages/careaware) and [Cerner Millennium](https://www.cerner.com/solutions/health-systems) data sources, as well as, understand complexity to better gauge what functions to bring into the solution. This was created using [Node-RED](https://nodered.org/) (created a mock service that published data to the app) and React (the application UI).

cchesser replied:
Note, I don't think we need to include hyperlinks to all the technologies, especially for ones which are commonly referenced. I included Node-RED, as I see that as not as common.

amcginn replied:
Cleaned up wording and included suggested links.

cchesser commented on source/_posts/2019-03-08-shipit-xiv.markdown:
```diff
@@ -0,0 +1,53 @@
+  </table>
+</div> 	
+
+Thank you to our talented judges Ben York, Adilson Ribeiro, and Scott Julius. Thank you to Kyle Harper for taking awesome pictures throughout our event.
```
Also, thank you to [Kyle Harper](https://twitter.com/kylesharpe) for taking awesome pictures throughout our event.

Include the link to Kyle's Twitter handle, where he was posting photos.

amcginn replied:
Included link to Kyle's Twitter

amcginn commented:
Addressed comments:
![shipit xiv - engineering health - 1](https://user-images.githubusercontent.com/22381097/54045715-f2bf4800-4197-11e9-9e34-9e8807756a2e.png)

cchesser commented:
I think the changes look good.
